js/web/docs/webnn-operators.md

```diff
@@ -89,7 +89,7 @@ operators and the supported opset domain/versions in **WebNN EP** by ONNX Runtim
 | ScatterElements | ai.onnx(11-12, 13-15, 16-17, 18+) | scatterElements | ✗ | ✓ | Only supports 'reduction' == 'none' |
 | ScatterND | ai.onnx(11-12, 13-15, 16-17, 18+) | scatterND | ✗ | ✓ | Only supports 'reduction' == 'none' |
 | Shape | ai.onnx(7-12, 13-14, 15-18, 19-20, 21+) | slice | ✓ | ✓ | |
-| SimplifiedLayerNormalization | com.microsoft(1+) | pow, reduceMean, add, sqrt, div, mul | ✓ | ✓ | |
+| SimplifiedLayerNormalization | ai.onnx(1+) | pow, reduceMean, add, sqrt, div, mul | ✓ | ✓ | |
```

Comment:
Is there an active ONNX CR for this? Otherwise SLN is Microsoft-specific (not an ONNX operator). https://github.com/onnx/onnx/blob/main/docs/Operators.md

Reply:
I didn't find it from https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md, even I didn't find its spec definition.

However in cpu ep, it is registered as onnx domain, https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cpu/cpu_contrib_kernels.cc#L141-L143, that's why I did this change.

Reply:
> However in cpu ep, it is registered as onnx domain

🤨 Typos happen. We probably shouldn't propagate this one, as SimplifiedLayerNormalization is not in ONNX.

FYI @amarin16 and @jingyanwangms.

Reply:
Here's the [feature request](https://github.com/onnx/onnx/issues/6582) to support SimplifiedLayerNormalization in ONNX.

Reply:
The original ORT mistake was 4 years ago (https://github.com/microsoft/onnxruntime/commit/20c47ce91c0fe6ce01ad37f4b8386a19a699753b) whereas that issue was opened 2 weeks ago, and ONNX operators can take months to be approved (and in some cases even be reverted or deprecated). So it's better to use the ai.onnx domain when the ONNX spec officially adds it.

However... if we expect it will go it, or the models using SkipSimplifiedLayerNorm specify the node domain as "" or "ai.onnx", then let's go with it; and if so, I approve (though, it's still technically a bug 😉). Can you verify what domain the nodes in the model refer to?

Reply:
In ONNX Runtime, it requires the node domain as “” or "ai.onnx", while in ONNX shape inference and model checker, it complains the `SkipSimplifiedLayerNorm` is not part of domain ai.onnx. 🤣

![image](https://github.com/user-attachments/assets/5ee54120-7654-4cc8-8285-30e330d768ad)

Reply:
Good, so the checker rightly complains 😜, but since we already have defective models out there using the incorrect domain, let's go ahead for compat.